subscriptions/tests.py

```diff
@@ -2013,46 +2005,31 @@ def test_metrics_read(self):
         # Check
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+        self.maxDiff = None
+        for r in sorted(response.data['metrics_available']):
+            print(r)
```

Comment:
You can probably remove this print and I can't see where `self.maxDiff` is used?

Reply:
Whoops, that was just me debugging the tests, forgot to remove. `maxDiff` allows you to see the full diff from `assertEqual`